.evg.yml

```diff
@@ -319,7 +329,7 @@ functions:
       working_dir: mongo-jdbc-driver
       script: |
           ${PREPARE_SHELL}
-          ./gradlew clean -x test -x integrationTest spotlessApply build --rerun-tasks
+          ./gradlew -Dorg.gradle.java.home=${JAVA_HOME} clean -x test -x integrationTest spotlessApply build --rerun-tasks
```

Comment:
The `amazon2-arm64-graviton3` host wasn't picking up the `$JAVA_HOME` variable in gradle automatically.  Setting it explicitly here works.